pkg/webui/console/containers/device-importer/index.js

```diff
@@ -212,38 +308,121 @@ export default class DeviceImporter extends Component {
   }
 
   get processor() {
-    const { log, totalDevices, devicesComplete, status, step, error } = this.state
+    const {
+      log,
+      devicesProcessed,
+      deviceErrors,
+      status,
+      step,
+      error,
+      convertedDevices,
+      aborted,
+    } = this.state
     const hasErrored = status === 'error'
     const { redirectToList } = this.props
     const operationMessage = step === 'conversion' ? m.converting : m.creating
     let statusMessage = m.operationInProgress
     if (status === 'error') {
       statusMessage = m.operationHalted
-    } else if (status === 'finished') {
+    } else if (status === 'finished' && !aborted) {
       statusMessage = m.operationFinished
+    } else {
+      statusMessage = m.operationAborted
```

Comment:
Not sure if the issue is related to this line, but this how the UI looks for me when end device import is in progress. I dont think we should show `Operation aborted` in this case

<img width="970" alt="Screenshot 2021-08-16 at 14 48 16" src="https://user-images.githubusercontent.com/16374166/129561988-1d767e35-eed8-436b-ae49-99b3f3e59b54.png">